x-pack/plugins/fleet/README.md

```diff
@@ -60,17 +60,17 @@ _The following is adapted from the Fleet Server [README](https://github.com/elas
 server.host: 0.0.0.0
 xpack.fleet.agents.enabled: true
 xpack.fleet.packages:
-    - name: fleet_server
```

Comment:
Looks like Prettier fired on this file when I committed my changes. I can revert these changes if we prefer.